values/prometheus-operator/prometheus-operator.gotmpl

```diff
@@ -174,8 +174,7 @@ grafana:
     securityContext: 
       runAsUser: 1000
     
-  plugins:
-    - grafana-piechart-panel
+  plugins: []
```

Comment:
is this the solution? Did you check all the grafana charts? I bet some are not rendering any more

Reply:
yes, the  grafana-piechart-panel is already part of grafana